### PR TITLE
WH-135: Admin Functionality for Teams, Universities, Games and Users

### PR DESCRIPTION
**Pull request by aag7593**
A shitload of stuff

- [x] Modification to the nav bar
- [x] Modification to the header
- [x] Creation of table element for admin display of data
- [x] Creation of a popup element
- [x] Manage teams page
- [x] Manage universities page
- [x] Manage games page
- [x] Manage users page
- [x] Add university
- [x] Update university
- [x] Delete university
- [x] Add user
- [x] Update user
- [x] Delete user
- [ ] Add user to team

### CODE REVIEW

**Comment by vsoler27** [comment]
im begging u to finish this so we can merge

File: src/components/navbar/navbar.js [hunk 1]
```diff
@@ -66,9 +66,37 @@ export default function Navbar() {
...
+                        <button className={styles.dropbtn}>
+                        {t("teamsanduniversitiesNav.label")}
+                          <FaCaretDown />
+                        </button>
+                      </th>
+                    </tr>
+                    <tr className={styles.dropcontent}>
+                      <td><a href="/manageTeams">TEAMS</a></td>
+                      <td><a href="/manageUniversities">UNIVERSITIES</a></td>
+                    </tr>
```
Comment (abp6318) [inline src/components/navbar/navbar.js:92]: Can we add /teamsanduniversities to this? That way admins can still view the `team/:id` pages should they want to?

File: src/components/teamblock/teamblock.js [hunk 2]
```diff
@@ -5,26 +5,27 @@ import globalStyles from '../../pages/pages.module.css';
 // import Team from '../../pages/teamsanduniversities/team/team';
 
 const TeamBlock = (props) => {
```
Comment (abp6318) [inline src/components/teamblock/teamblock.js:7]: Remove any commented code if possible

**Review by abp6318 — CHANGES_REQUESTED** [review]
Really fucking good work. Few small things to look at; removing console logs and comments is low priority imo.

Amazing work girly! Vicky and I are geeking out at how good this all looks :)

File: src/components/editpopup/editpopup.js [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+                                id="team" 
+                                name="team" 
+                                placeholder='Select Team Name' 
+                                value={teamSelected.description}  
+                                disabled
+                            />
+                        </div>
+
+                        <select size="3" className={styles.dropdown} onChange={(e) => handleTeamClick(e)}>
+                            <option key={0} value={JSON.stringify(nothing)}>{nothing.description}</option>
```
[the remark this answers is not in the document]
Comment (vsoler27) [reply src/components/editpopup/editpopup.js:341]: same thing as null methinks

File: src/components/editpopup/editpopup.js [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+    }
+
+    async function onSubmitUser(e) {
+        let myHeaders = new Headers();
+        myHeaders.append("Content-Type", "application/json");
+
+         const raw = JSON.stringify({
+            "id": props.data._id,
+            "roleID": e.target.roleID.value,
+            "teamID": teamSelected._id,
```
[the remark this answers is not in the document]
Comment (aag7593) [reply src/components/editpopup/editpopup.js:111]: Ah shit I knew I missed a part- I can look at that tomorrow morning

**Review by abp6318 — APPROVED** [review]
slayed -- making changes in another PR so we can get this stuff in main

**Pull request merged**